lib/modules/manager/gomod/extract.spec.ts

```diff
@@ -13,11 +14,11 @@ describe('modules/manager/gomod/extract', () => {
     it('extracts single-line requires', () => {
       const res = extractPackageFile(gomod1)?.deps;
       expect(res).toMatchSnapshot();
-      expect(res).toHaveLength(9);
+      expect(res).toHaveLength(10);
```

Comment:
What causes the difference?

Reply:
the local deps are now extracted, see snapshot update above